cpp/velox/memory/VeloxColumnarBatch.h

```diff
@@ -46,6 +46,10 @@ class VeloxColumnarBatch final : public ColumnarBatch {
   facebook::velox::RowVectorPtr getRowVector() const;
   facebook::velox::RowVectorPtr getFlattenedRowVector();
 
+  static facebook::velox::RowVectorPtr convertBatch(
+      std::shared_ptr<facebook::velox::memory::MemoryPool> pool,
+      std::shared_ptr<ColumnarBatch> cb);
+
```

Comment:
Can we declare this out from the class scope?

Reply:
Because it converts batch to like VeloxColumnarBatch, so I put it here.
Do you think it should exist in VeloxColumnarBatch.h but not in VeloxColumnarBatch class?

Reply:
Fixed, can you help review again?